cli.ts

```diff
@@ -69,7 +69,7 @@ async function generate() {
 
 try {
   await Deno.remove(metafile);
-} catch (e) {
+} catch (_e) {
```

Comment:
```suggestion
} catch {
```

We don't need to do `_e` here.